.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
A bunch of IntelliJ files have snuck in here. 

Idk whether we should choose to add them to the .gitignore, or whether everyone should do https://sebastiandedeyne.com/setting-up-a-global-gitignore-file

Reply:
oh gosh how embarassing